routes/breadcrumbs.php

```diff
@@ -333,7 +333,7 @@
 Breadcrumbs::for('archive', function (BreadcrumbTrail $trail): void
 {
     $trail->parent('home');
-    $trail->push('Visit Us',route('visit'));
+    $trail->push('Visit Us', '/plan-your-visit');
```

Comment:
Why was this changed? Surely using the `route` method is more reliable than hard coding the URL like this?

Reply:
Hi @sean-dunwoody for some reason when I worked on this yesterday my local one didn't like `route('visit') when I tried preview all archive.

I tested this again and it works fine, so I removed this.

Thanks for spotting this